docs/source/contents/servers/index.md

```diff
@@ -22,6 +22,20 @@ There is a default capabilities for each server as follows:
   :end-before: SELDON_OVERCOMMIT_PERCENTAGE
   ```
 
+## Extra Capabilities
+Servers can be annotated with the `extraCapabilities` field to indicate custom configurations (e.g. Python dependencies). For instance:
```

Comment:
Given this is a k8s resource, probably best to avoid `annotated` as that implies `metadata.annotations`

Reply:
Agreed. Fixed wording in https://github.com/SeldonIO/seldon-core/pull/4718/commits/1e7bb30d7492d9b4fbd56f8de232f13cb2400b9f.